### PR TITLE
Rename wrapper

### PR DESCRIPTION
**Pull request by i-am-yuvi**
## Describe your changes
The `wrapper` field is renamed to `eventer` in the same way `wrap` is changed to `event`.

## Issue ticket number and link
#18 

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have compiled and built the program.

### CODE REVIEW

**Comment by 0xB10C** [comment]
feel free to squash the `cargo fmt` changes with the rename commit :)

**Comment by i-am-yuvi** [comment]
> feel free to squash the `cargo fmt` changes with the rename commit :)

Done!!!

File: protobuf/python-types/event_msg_pb2.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# source: wrapper.proto
+# source: event_msg.proto
```
Comment (0xB10C) [inline protobuf/python-types/event_msg_pb2.py:3]: not a big problem, but I think this should be in the second commit (no need to change it).

**Review by 0xB10C — COMMENTED** [review]
Can you tell me what `protoc --version` outputs for you? It's `libprotoc 24.4` for me and the diff of the python protobuf files is a lot smaller when running `gen-python.sh`. Wondering if we are using the same tool/version?

**Comment by i-am-yuvi** [comment]
> Can you tell me what `protoc --version` outputs for you? It's `libprotoc 24.4` for me and the diff of the python protobuf files is a lot smaller when running `gen-python.sh`. Wondering if we are using the same tool/version?

Okay my bad the version of protoc in my system is 3.1 I had downloaded from apt, I have downloaded the new one from github release.
Will update the changes with the typo and squash it with the second commit, I think this won't be an issue?

File: protobuf/python-types/mempool_pb2.py [hunk 2]
```diff
@@ -3,31 +3,361 @@
...
+  serialized_pb=b'\n\rmempool.proto\x12\x07mempool\"\xab\x01\n\x0cMempoolEvent\x12\x1f\n\x05\x61\x64\x64\x65\x64\x18\x01 \x01(\x0b\x32\x0e.mempool.AddedH\x00\x12%\n\x08replaced\x18\x02 \x01(\x0b\x32\x11.mempool.ReplacedH\x00\x12#\n\x07removed\x18\x03 \x01(\x0b\x32\x10.mempool.RemovedH\x00\x12%\n\x08rejected\x18\x04 \x01(\x0b\x32\x11.mempool.RejectedH\x00\x42\x07\n\x05\x65vent\"1\n\x05\x41\x64\x64\x65\x64\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\r\n\x05vsize\x18\x02 \x02(\x05\x12\x0b\n\x03\x66\x65\x65\x18\x03 \x02(\x03\"W\n\x07Removed\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\x0e\n\x06reason\x18\x02 \x02(\t\x12\r\n\x05vsize\x18\x03 \x02(\x05\x12\x0b\n\x03\x66\x65\x65\x18\x04 \x02(\x03\x12\x12\n\nentry_time\x18\x05 \x02(\x04\"(\n\x08Rejected\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\x0e\n\x06reason\x18\x02 \x02(\t\"\xba\x01\n\x08Replaced\x12\x15\n\rreplaced_txid\x18\x01 \x02(\x0c\x12\x16\n\x0ereplaced_vsize\x18\x02 \x02(\x05\x12\x14\n\x0creplaced_fee\x18\x03 \x02(\x03\x12\x1b\n\x13replaced_entry_time\x18\x04 \x02(\x04\x12\x18\n\x10replacement_txid\x18\x05 \x02(\x0c\x12\x19\n\x11replacement_vsize\x18\x06 \x02(\x05\x12\x17\n\x0freplacement_fee\x18\x07 \x02(\x03'
+)
```
Comment (0xB10C) [inline protobuf/python-types/mempool_pb2.py:23]: I'm asking as I wouldn't expect e.g. `mempool_pb2.py` (and some of the other files) to change when renaming something unrelated.

**Review by 0xB10C — COMMENTED** [review]
I'm not entirely sure which `protoc` tool you are using. Can you point me to the GitHub release you've downloaded? It seems we are using two different tools.

When we're not using the same tool, the next time I'm going to update the python-types with my `protoc` tool this will change back. Ideally, I want to avoid that.

**Comment by i-am-yuvi** [comment]
> I'm not entirely sure which `protoc` tool you are using. Can you point me to the GitHub release you've downloaded? It seems we are using two different tools.
> 
> When we're not using the same tool, the next time I'm going to update the python-types with my `protoc` tool this will change back. Ideally, I want to avoid that.

I am using `libprotoc 25.1` in my machine.

**Comment by i-am-yuvi** [comment]
@0xB10C I am getting the size of python file very less when running with `libprotoc v24.4` as compared to other versions of `libprotoc`. Shall I push that??

**Comment by 0xB10C** [comment]
> @0xB10C I am getting the size of python file very less when running with `libprotoc v24.4` as compared to other versions of `libprotoc`. Shall I push that??

Yes, please try that. Using v25 or v26 still produces files matching what's on master for me.

**Comment by i-am-yuvi** [comment]
I guess it looks fine now!!

**Comment by 0xB10C** [comment]
Awesome, yes! Thanks!

**Comment by 0xB10C** [comment]
We'll probably need to update some of the python tools too - they aren't tested by CI yet. Not here, though :)

**Pull request merged**